less/variables.less

```diff
@@ -24,3 +24,7 @@
 @border-real: #dee2e6;
 
 // Mobile var
+
+@phone: ~"(max-width: 500px)";
+
+@iphone5s: ~"(max-width: 400px)";
```

Comment:
very interesting.  so you're doing media queries specifically for the iphone.  neat!